publish-service/src/main/resources/application.properties

```diff
@@ -5,12 +5,14 @@ debug: false
 spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
 
 #Logging configurations 
-logging.level.root: INFO
-logging.level.org.springframework.web: INFO
-logging.level.com.ericsson.eiffel.remrem.producer: INFO
+logging.level.root: DEBUG
```

Comment:
We can change the log-level to info, as this app.prop file is for our local run purpose...

Reply:
Done